src/translator_testing_model/schema/translator_testing_model.yaml

```diff
@@ -0,0 +1,479 @@
+---
+id: https://w3id.org/TranslatorSRI/test-case-schema
+name: Translator-Testing-Model
+title: Translator Testing Data Model
+description: |-
+  Data model to formalize the structure of test assets, cases, suites and related metadata
+  applied to run the diverse polymorphic testing objectives for the Biomedical Data Translator system.
+license: MIT
+see_also:
+  - https://TranslatorSRI.github.io/test-case-schema
+
+prefixes:
+  ttm: https://w3id.org/TranslatorSRI/Translator-Testing-Model/
+  linkml: https://w3id.org/linkml/
+  biolink: https://w3id.org/biolink/
+  schema: http://schema.org/
+  example: https://example.org/
+
+default_prefix: ttm
+default_range: string
+
+imports:
+  - linkml:types
+
+classes:
+
+  TestEntity:
+    description: >-
+      Parent identification class of all major classes within the data model for Translator testing.
+    abstract: true
+    tree_root: true
+    slots:
+      - id
+      - name
+      - description
+
+  TestMetadata:
+    description: >-
+      Represents metadata related to (external SME, SMURF, Translator feedback, 
+      large scale batch, etc.) provenance of test assets, cases and/or suites.
+    is_a: TestEntity
```

Comment:
is TestMetadata provided as a member of the `inputs` slot?

Reply:
Not really. The TestMetadata is mainly for provenance and cross-referencing (i.e. Github issue citations?) of the source of a given set of contributed test assets/cases/suites into the repository, that is, who, what and from where were these contributed? As such, I suspect that it is mainly used at the TestSuite level (although perhaps individual TestAsset or TestCase instances could have pointers back to such metadata(?)... need to see what we need to do in practice).